tests/testthat/test-selections.R

```diff
@@ -0,0 +1,213 @@
+library(testthat)
+library(recipes)
+library(tibble)
+library(tidyselect)
+library(rlang)
```

Comment:
I feel quite strongly that tibble, tidyselect, and rlang should not be `library()`ied here. That makes them available to all tests, which goes against the idea that this test file should be self contained. Just prefix their calls with `rlang::` etc, or if we have done `@import rlang` then you don't need this at all.

I personally don't think testthat or recipes should be libraried either, as `load_all()` and `check()` always loads them for you. But if Max needs them, then that is fine